clouds/clouds-core/src/main/java/sunstone/core/SunstoneExtension.java

```diff
@@ -69,10 +72,44 @@ public void beforeAll(ExtensionContext ctx) throws Exception {
         if (ctx.getRequiredTestClass().getAnnotationsByType(WithAwsCfTemplate.class).length > 0) {
             handleAwsCloudFormationAnnotations(ctx);
         }
+        if (ctx.getRequiredTestClass().getAnnotationsByType(Setup.class).length > 0) {
+            handleSetup(ctx);
+        }
         performDeploymentOperation(ctx);
         injectStaticResources(ctx, ctx.getRequiredTestClass());
     }
 
+    static void handleSetup(ExtensionContext ctx) throws IllegalArgumentSunstoneException {
+        SunstoneStore store = new SunstoneStore(ctx);
+        if (ctx.getRequiredTestClass().getAnnotationsByType(Setup.class).length != 1) {
+            throw new RuntimeException();
```

Comment:
I think some error message should be present.

Reply:
Fixed.